build/images/scripts/start_ovs_netdev

```diff
@@ -35,6 +35,7 @@ function add_br_phy {
     ip addr flush dev eth0 2>/dev/null
     ip link set eth0 up
     ip route add default via "$gw" dev br-phy
+    iptables -t raw -A PREROUTING -i eth0 -j DROP
```

Comment:
Do we need to do some cleanup before we append it to avoid errors or appending duplicate rules?

Reply:
let me run some tests, it seems that the same issue should exist with the `ip addr add` command, which should fail as well if the IP address already exists on the interface

Reply:
I have updated the script to handle container restarts, which wasn't the case until now